qutebrowser/browser/browsertab.py

```diff
@@ -759,6 +760,7 @@ def __init__(self, *, win_id, mode_manager, private, parent=None):
         self._mouse_event_filter = mouse.MouseEventFilter(
             self, parent=self)
         self.backend = None
+        self._last_hovered_link = None
```

Comment:
Data attached to a tab like that should be moved to the respective `TabData` object (see `browsertab.py`).